specification/resources/resource-manager/Microsoft.Solutions/stable/2018-02-01/examples/createOrUpdateApplication.json

```diff
@@ -0,0 +1,43 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg",
+    "managedApplicationName": "myManagedApplication",
+    "api-version": "2018-02-01",
+    "parameters": {
+      "properties": {
+        "managedApplicationDefinitionId": "/subscriptions/subid/resourceGroups/rg/providers/Microsoft.Solutions/applicationDefinitions/myAppDef",
+        "managedResourceGroupId": "/subscriptions/subid/resourceGroups/myManagedRG"
+      },
+      "name": "myManagedApplication",
+      "location": "East US 2",
+      "kind":  "ServiceCatalog"
+    }
+  },
+  "responses": {
+    "201": {
+      "headers": {},
+      "body": {
```

Comment:
Similar to getApplication.json - the response here does not match the definition of `Application`.